src/manifest.firefox.json

```diff
@@ -0,0 +1,34 @@
+{
+  "manifest_version": 2,
+
+  "name": "Markdown to JIRA",
+  "description": "Converts Markdown to JIRA wiki syntax",
+
+  "applications": {
```

Comment:
This is a key that's required for Firefox, especially because we're using browser storage, but is not supported by Chrome.